addons/resource.language.en_gb/resources/strings.po

```diff
@@ -9817,15 +9820,15 @@ msgid "Login screen"
 msgstr ""
 
 msgctxt "#20097"
-msgid "Fetching album info"
+msgid "Fetching album information"
 msgstr ""
 
 msgctxt "#20098"
-msgid "Fetching info for album"
+msgid "Fetching information for album"
 msgstr ""
 
 msgctxt "#20099"
-msgid "Can't rip CD or track while playing from CD"
+msgid "Can not rip a CD or track while playing from the CD"
```

Comment:
Using **cant't**, **cannot**, **can not** are all correct. I found a reference to error messages on software that use can't, so can't it is.